source/command-monitoring/command-monitoring.rst

```diff
@@ -179,25 +123,10 @@ A document sequence in an OP_MSG reply to "aggregate" or "getMore" MUST be simil
 
 See "Why are document sequences included as BSON arrays?" in the `rationale`_.
 
-Read Preference
-^^^^^^^^^^^^^^^
-
-In cases where queries or commands are embedded in a ``$query`` parameter when a read preference
-is provided, they MUST be unwrapped and the value of the ``$query`` attribute becomes the
-``filter`` or the command in the started event. The read preference will subsequently be dropped
-as it is considered metadata and metadata is not currently provided in the command events.
-
 ---------
 Rationale
 ---------
 
-*1. Why does the specification treat all events as commands, even those that are not sent as such?*
```

Comment:
These aren't prose tests, so I don't see a problem with re-numbering after you remove some questions. Alternatively, just remove the numbers entirely and leave the questions on their own. These are just emphasized text and not linked headers so we need not worry about anything linking directly to these.

Reply:
good catch. renumbered